specfile: fedora review changes

supakeen opened the pull request:
Changes as pointed out in the [Fedora package review](https://bugzilla.redhat.com/show_bug.cgi?id=2339108).

ondrejbudai reviewed (changes requested):
For the record, the only concerning thing about this PR is that the `Source0` is now hardcoded to bundled releases (as required by the Fedora Packaging Guideline).

We were wondering about how packit handles it in its COPR integration. It seems like it doesn't actually care about the original `Source0`, and it just overrides it with whatever is needed to build a branch/PR. I verified this by downloading the SRPM, and inspecting the embedded tarball.

Just one thing: @supakeen, can you try removing the version change? In all other projects, we have LATEST_VERSION+1 in the specfile on main, so it would be great to do this here as well (otherwise we would have to adjust the automation I think).

supakeen commented:
> For the record, the only concerning thing about this PR is that the `Source0` is now hardcoded to bundled releases (as required by the Fedora Packaging Guideline).
> 
> We were wondering about how packit handles it in its COPR integration. It seems like it doesn't actually care about the original `Source0`, and it just overrides it with whatever is needed to build a branch/PR. I verified this by downloading the SRPM, and inspecting the embedded tarball.
> 
> Just one thing: @supakeen, can you try removing the version change? In all other projects, we have LATEST_VERSION+1 in the specfile on main, so it would be great to do this here as well (otherwise we would have to adjust the automation I think).

Of course; I've rebased and changed the `Source0` commit to drop the version change which now puts it on the same version as `main` again.

ondrejbudai reviewed (approved):
Thanks!
